ibm/data_source_ibm_is_ike_policy.go

```diff
@@ -0,0 +1,311 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+)
+
+func dataSourceIBMIsIkePolicy() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMIsIkePolicyRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ExactlyOneOf: []string{"name", "ike_policy_id"},
+				Description:  "The IKE policy name.",
+			},
+			"ike_policy_id": {
```

Comment:
Can you please just rename to ike_policy just to be line with all other resources